### PR TITLE
README recomments conda as inatall method

### PR DESCRIPTION
**Pull request by hamogu**
and I'm sure that's fine, but it required conda-forge in the conda channels.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -35,6 +35,8 @@ be done easily using conda or pip::
   conda install cookiecutter
```
Comment (Cadair) [inline README.rst]: it might be better to change this line to:

`conda install -c conda-forge cookiecutter gitpython`

File: README.rst [hunk 2]
```diff
@@ -35,6 +35,8 @@ be done easily using conda or pip::
...
 
   pip install cookiecutter
+  
+(The conda method only works if `conda-forge` is in your default channels. Use ``conda config --add channels conda-forge`` to add this channel.)
```
Comment (Cadair) [inline README.rst]: it's probably better to recommend `config --append channels conda-forge`

**Comment by hamogu** [comment]
changes according to @Cadair suggestions

File: README.rst [hunk 3]
```diff
@@ -32,10 +32,11 @@ This package template makes use of the `cookiecutter
 easier to get started with the package template. You will need to `install cookiecutter <https://cookiecutter.readthedocs.io/en/latest/installation.html>`__ which can
 be done easily using conda or pip::
 
-  conda install cookiecutter
+  conda install -c conda-forge cookiecutter gitpython
 
   pip install cookiecutter
```
Comment (Cadair) [inline README.rst]: This command should have gitpython as well now I think about it.

**Pull request merged**